kitops.rb.template

```diff
@@ -1,9 +1,9 @@
 class Kitops < Formula
   desc "Packaging and versioning system for AI/ML projects"
   homepage "https://KitOps.ml"
-  license "Apache-2.0"
   version "@@version"
- 
+  license "Apache-2.0"
```

Comment:
For what it's worth, the last update PR we generated did not include the version field -- not sure why.